spec/models/social_networking/goal_spec.rb

```diff
@@ -48,5 +48,18 @@ module SocialNetworking
 
       expect(Goal.for_week.count).to eq(count + 1)
     end
+
+    context "with goal sample data" do
+      fixtures(:all)
+
+      it "should return an incomplete goal" do
+        expect(Goal.did_not_complete.count).to eq(1)
+      end
+
+      it "should return no incomplete goals" do
```

Comment:
@eschlange I forgot to say this, but we should consider updating this to this to something like:
```
expect { Goal.find_by(description: "p1 omega").delete }.to change(Goal.for_week.count, :count).by(-1)
```
See guide note: https://github.com/cbitstech/guides/tree/master/testing/rspec#avoid-using-incidental-state